lib/maril/generator.rb

```diff
@@ -7,7 +7,7 @@ def initialize(options={})
 
     def generate
       app = fetch_app
-      cmd = [ 'docker run' ]
+      cmd = [ 'docker run --rm -it' ]
```

Comment:
I had thought before to add these options into the command. But there are lots of cases that we don't want them. Specially when we trying to debug things that crash app. So you have to build image from a stopped container. The remove option will remove the container after it stop. So do you think we should use command option to add them?

Then it lead another question. How about the browser extension?

I don't have a strong opinion of this. If you think there will be 60% cases that we remove the container right after it stop. Then just leave it like this.